src/generate_initial_weekly_plots.R

```diff
@@ -102,7 +102,8 @@ generate_initial_weekly_plots <- function(all_df_list, pending_df_list, site_arg
             # remove empty dfs from the list
             keep(~ nrow(.)>0) %>%
             bind_rows() %>%
-            arrange(weekday)
+            arrange(day)
```

Comment:
inspect some worked correctly for me when testing! strange bug tho

Reply:
Yes, a very weird - maybe {lubridate} - bug?